server/src/internalClusterTest/java/org/elasticsearch/search/morelikethis/MoreLikeThisIT.java

```diff
@@ -289,14 +288,8 @@ public void testMoreLikeThisIssue2197() throws Exception {
         indicesAdmin().prepareRefresh("foo").get();
         assertThat(ensureGreen(), equalTo(ClusterHealthStatus.GREEN));
 
-        SearchResponse response = client().prepareSearch()
-            .setQuery(new MoreLikeThisQueryBuilder(null, new Item[] { new Item("foo", "1") }))
-            .get();
-        assertNoFailures(response);
-        assertThat(response, notNullValue());
```

Comment:
Removing asserting non-null in a couple spots like this, no clue why that was even there to begin with, the other assertion would NPE already for a null `response`